pnet_macros/src/decorator.rs

```diff
@@ -1154,6 +1154,8 @@ fn parse_ty(ty: &str) -> Option<(usize, Endianness, EndiannessSpecified)> {
             let e = e.as_str();
             if e == "be" {
                 (Endianness::Big, EndiannessSpecified::Yes)
+            } else if e == "he" {
```

Comment:
I'd like to see some test cases added for this please.

Reply:
Just added one test case for this, as the other also don't have more. Is that okay?